ReactAndroid/src/androidTest/js/ScrollViewTestModule.js

```diff
@@ -47,90 +46,84 @@ var getInitialState = function() {
   };
 };
 
-var onScroll = function(e) {
+const onScroll = function(e) {
   ScrollListener.onScroll(
     e.nativeEvent.contentOffset.x,
     e.nativeEvent.contentOffset.y,
   );
 };
 
-var onScrollBeginDrag = function(e) {
+const onScrollBeginDrag = function(e) {
   ScrollListener.onScrollBeginDrag(
     e.nativeEvent.contentOffset.x,
     e.nativeEvent.contentOffset.y,
   );
 };
 
-var onScrollEndDrag = function(e) {
+const onScrollEndDrag = function(e) {
   ScrollListener.onScrollEndDrag(
     e.nativeEvent.contentOffset.x,
     e.nativeEvent.contentOffset.y,
   );
 };
 
-var onItemPress = function(itemNumber) {
+const onItemPress = function(itemNumber) {
   ScrollListener.onItemPress(itemNumber);
 };
 
-var ScrollViewTestApp = createReactClass({
-  displayName: 'ScrollViewTestApp',
-  getInitialState: getInitialState,
-  onScroll: onScroll,
-  onItemPress: onItemPress,
-  onScrollBeginDrag: onScrollBeginDrag,
-  onScrollEndDrag: onScrollEndDrag,
+class ScrollViewTestApp extends React.Component {
+  scrollView = React.createRef();
+  state = getInitialState();
 
-  scrollTo: function(destX, destY) {
-    this.refs.scrollView.scrollTo(destY, destX);
-  },
+  scrollTo = (destX, destY) => {
+    this.scrollView.scrollTo(destY, destX);
```

Comment:
I should have caught this earlier, but when you create a ref using `React.createRef`, you have to access what it's referring to using `ref.current`. So, this should be something like:

```JavaScript
const { scrollView: { current: scrollView } } = this;
if (scrollView == null) {
  return;
}
scrollView.scrollTo(destY, destX);
```

But don't worry about it. 😄 I'll make the fix internally.